api/src/main/resources/ucum-fhir-essence.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+-->
+<root version="1.0" revision="$Revision: 1 $" revision-date="$Date: 2022-06-21 16:00:00 -0500 (Tue, 21 June 2022) $">
+    <unit Code="mmHg" CODE="MMHG" isMetric="yes" class="clinical">
```

Comment:
@ibacher let me know if there are any other custom definitions we need to add

Reply:
Heh... I think that's something we'll have to figure out from practice. This is a good starting point though!